src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpContentJsonExtensions.AsyncEnumerable.cs

```diff
@@ -60,17 +60,18 @@ public static partial class HttpContentJsonExtensions
             return ReadFromJsonAsAsyncEnumerableCore<TValue>(content, options, cancellationToken);
         }
 
-        [RequiresUnreferencedCode(SerializationUnreferencedCodeMessage)]
-        [RequiresDynamicCode(SerializationDynamicCodeMessage)]
-        private static IAsyncEnumerable<TValue?> ReadFromJsonAsAsyncEnumerableCore<TValue>(
-            HttpContent content,
-            JsonSerializerOptions? options,
-            CancellationToken cancellationToken)
-        {
-            var jsonTypeInfo = (JsonTypeInfo<TValue>)JsonHelpers.GetJsonTypeInfo(typeof(TValue), options);
-            return ReadFromJsonAsAsyncEnumerableCore(content, jsonTypeInfo, cancellationToken);
-        }
-
+        /// <summary>
+        /// Reads the HTTP content and returns the value that results from deserializing the content as
+        /// JSON in an async enumerable operation.
+        /// </summary>
+        /// <typeparam name="TValue">The target type to deserialize to.</typeparam>
+        /// <param name="content">The content to read from.</param>
+        /// <param name="jsonTypeInfo">Source generated JsonTypeInfo to control the behavior during deserialization.</param>
```

Comment:
Must it be "source generated"?

Reply:
I suppose not, I can update the existing places that use the same wording.